tests/test_github_actions_poll_bound_current_head_regressions.py

```
@@ -0,0 +1,186 @@
+
+
+def _renamed_transport_branch(command: str = "gh api repos/example/repo/pulls/7/reviews") -> str:
+    """Return the reviewed renamed transport-failure branch."""
+    return f'''  if ! response="$(%s)"; then
+    api_error_streak=$((api_error_streak + 1))
+    if [ "$api_error_streak" -ge "$transport_error_budget" ]; then
+      exit 1
+    fi
+    continue
+  fi
+  api_error_streak=0''' % command
```

Ruff (0.16.3)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)